testsuite/MDAnalysisTests/test_transformations.py

```diff
@@ -238,7 +239,7 @@ def test_projection_from_matrix_4(self):
         assert_equal(t.is_same_transform(P0, P1), True)
 
 
-class _ClipMatrix(object):
+class _ClipMatrix(unittest.TestCase):
```

Comment:
It provides access to the `unittest` module `self.assert` type statements through inheritance.